docs/resources/tutorials/cloud/router_participant_security_via_xml_profiles/configurations/secure_domain1.xml

```diff
@@ -13,32 +13,32 @@
                     <!-- Configure DDS:Auth:PKI-DH plugin -->
                     <property>
                         <name>dds.sec.auth.builtin.PKI-DH.identity_ca</name>
-                        <value>file:///home/demo_keystore/enclaves/talker_listener/listener/identity_ca.cert.pem</value>
+                        <value>file:///~/sros2_demo/demo_keystore/enclaves/ros2_router/participant1/identity_ca.cert.pem</value>
```

Comment:
I would suggest to write the path as home/  because ~/ didn't work for me 
```suggestion
                        <value>file:///path/to/sros2_demo/demo_keystore/enclaves/ros2_router/participant1/identity_ca.cert.pem</value>
```